src/index.html

```diff
@@ -1,13 +1,322 @@
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles/styles.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">
+    <header class="header">
+      <div class="header__content">
+        <div class="top-bar">
+          <a href="#" class="top-bar__logo">
+            <img src="/images/Logo.svg" alt="Bike Logo">
+          </a>
+
+          <div class="top-bar__icons">
+            <a href="tel:+1 234 5555-55-55" class="icon icon--phone"></a>
+
+            <a href="#menu" class="icon icon--menu"></a>
+          </div>
+        </div>
+
+        <div class="header__bottom">
+          <h1 class="header__title">Take the Streets</h1>
+        </div>
+      </div>
+    </header>
+
+    <aside class="page__menu menu" id="menu">
+      <div class="top-bar menu__top">
+        <a href="#" class="top-bar__logo">
+          <img src="/images/logodark.png" alt="bike logo">
+        </a>
+
+        <a href="#" class="icon icon--close"></a>
+      </div>
+
+      <div class="menu__content">
+        <nav class="nav menu__nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link" href="#">Home</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#about-us">About Us</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#compare-bikes">Compare Bikes</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#details">Details</a>
+            </li>
+
+            <li class="nav__item">
+              <a class="nav__link" href="#contacts">Contacts</a>
+            </li>
+          </ul>
+        </nav>
+
+        <a href="tel:+1 234 5555-55-55" class="menu__phone-number">
+          +1 234 5555-55-55
+        </a>
+
+        <a href="tel:+1 234 5555-55-55" class="menu__book-a-test-ride">Book a test ride</a>
+      </div>
+    </aside>
+
+    <main class="main">
+      <section class="about-us" id="about-us">
+        <h2 class="about-us__title">Move Free</h2>
+
+        <article class="about-us__information">
+          Shift your ride, not gears.
+          Find the fastest way to move in the city as the bike adapts
+          intuitively to power the speed you need.
+        </article>
+      </section>
+
+      <section class="compare-bikes" id="compare-bikes">
+        <h2 class="section-title">Compare bikes</h2>
+
+        <div class="compare-bikes__bikes">
+          <article class="compare-bikes__product">
+            <a href="#">
+              <img
+              src="images/bike1.png"
+              alt="first-bike"
+              class="bike__photo"
+              >
+            </a>
+            <h3 class="title">Sporty 4</h3>
```

Comment:
Still, alt is not perfect)

I would use the actual bike article - ex. `Bike Sporty 4`

The reason for requiring perfect semantic tags and attributes is that landing is the most critical page from the SEO perspective and should get the most attention from this point

Reply:
It is not critical for now - so you may leave it as it is :)